plugins/src/android/optimove.gradle

```diff
@@ -35,4 +37,18 @@ buildscript {
         
         
     }
+}
+ext.postBuildExtras = {
```

Comment:
// use postBuildExtras to make sure the plugin is applied after
// cdvPluginPostBuildExtras. Therefore if googleServices is added
// to cdvPluginPostBuildExtras somewhere else, the plugin execution
// will be skipped and project build will be successfull

i think this comment is useful

Reply:
done :+1